validation-test/Driver/Inputs/clang_rt-helper.h

```diff
@@ -0,0 +1,9 @@
+#include <stdbool.h>
+
+static inline bool testNewOS() {
```

Comment:
Is this testing the OS, or testing the availability of the OS? If the latter, the current name could be improved.